app/src/main/java/org/akvo/flow/ui/fragment/ResponseListFragment.java

```diff
@@ -42,19 +42,18 @@
 import org.akvo.flow.R;
 import org.akvo.flow.activity.FormActivity;
 import org.akvo.flow.activity.TransmissionHistoryActivity;
-import org.akvo.flow.data.loader.SurveyInstanceLoader;
 import org.akvo.flow.data.database.SurveyDbAdapter;
+import org.akvo.flow.data.loader.SurveyInstanceLoader;
 import org.akvo.flow.domain.SurveyGroup;
-import org.akvo.flow.domain.SurveyedLocale;
 import org.akvo.flow.ui.adapter.ResponseListAdapter;
 import org.akvo.flow.util.ConstantUtil;
 
+import static org.akvo.flow.util.ConstantUtil.EXTRA_RECORD_ID;
+import static org.akvo.flow.util.ConstantUtil.EXTRA_SURVEY_GROUP;
+
 public class ResponseListFragment extends ListFragment implements LoaderCallbacks<Cursor> {
     private static final String TAG = ResponseListFragment.class.getSimpleName();
 
-    private static final String EXTRA_SURVEY_GROUP = "survey_group";
-    private static final String EXTRA_RECORD = "record";
-
     // TODO: Move all id constants to ConstantUtil
```

Comment:
yes it was made by Iñigo and is still relevant but it is used in 2 classes ResponseListFragment and ResponseListAdapter, I can do the changes but I think they are kind of out of scope here since I am mostly working on datapoints and looking at those 2 they will eventually also need refactoring so I would do that at that point, what do you think?